src/backend/InvenTree/common/apps.py

```diff
@@ -37,3 +42,24 @@ def clear_restart_flag(self):
                     set_global_setting('SERVER_RESTART_REQUIRED', False, None)
         except Exception:
             pass
+
+    def override_global_settings(self):
```

Comment:
Wouldn't it be saver / easier to check for this when settings are read? The proposed change could make the system quite hard to debug.

Reply:
I think this has great potential to confuse the user. The docs currently just state that one might override settings globally at 2 places and then point at each other in a circular fashion.
Maybe we need to restructure the settings docs to be more clear about how configuration values are read and how to debug them.

Reply:
I've made some improvements to the docs here